pkl-executor/src/main/java/org/pkl/executor/EmbeddedExecutor.java

```diff
@@ -157,7 +157,15 @@ private PklDistribution findCompatibleDistribution(
 
   private static Path toDisplayPath(Path modulePath, ExecutorOptions options) {
     var rootDir = options.getRootDir();
-    return rootDir == null ? modulePath : rootDir.relativize(modulePath);
+    return rootDir == null ? modulePath : relativize(modulePath, rootDir);
+  }
+
+  // Windows relativize will fail if the two paths have different roots.
+  private static Path relativize(Path path, Path base) {
+    if (path.isAbsolute() && base.isAbsolute() && !path.getRoot().equals(base.getRoot())) {
```

Comment:
I'd be inclined to push it in
```suggestion
    return relativize(modulePath, rootDir);
  }

  // Windows relativize will fail if the two paths have different roots.
  private static Path relativize(Path path, Path base) {
    if (base == null || path.isAbsolute() && base.isAbsolute() && !path.getRoot().equals(base.getRoot())) {
```

Reply:
I actually find the former a little easier to read, because of fewer binary operators grouped in the `if`